src/backend/joanie/core/filters/client.py

```diff
@@ -17,7 +17,7 @@ class OrderViewSetFilter(filters.FilterSet):
 
     product = filters.UUIDFilter(field_name="product")
     course = filters.CharFilter(field_name="course__code")
-    state = filters.ChoiceFilter(field_name="state", choices=enums.ORDER_STATE_CHOICES)
```

Comment:
there is a problem in your commit's message?
change order's state filter to accept multiple choices ?